app/models/ext_management_system.rb

```diff
@@ -27,6 +27,9 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
+  include ComplianceMixin
+  include CustomAttributeMixin
+  after_initialize :load_custom_attributes
```

Comment:
> @alongoldboim you probably want to move this functionality down to the container manager. If we need to expose this type of relationship for all managers, we can look at that later.

@alongoldboim @blomquisg idea was to bring this functionality to all type of providers. If this changed then it needs to be properly advertised to everyone (it's very important that it is going to be clear).